src/app/routes/index.test.jsx

```diff
@@ -21,13 +21,14 @@ import mediaAssetPageJson from '#data/yoruba/cpsAssets/media-23256797.json';
 import { ERROR_PAGE, FRONT_PAGE } from '#app/routes/utils/pageTypes';
 import * as fetchDataFromBFF from '#app/routes/utils/fetchDataFromBFF';
 import gahuzaOnDemandAudio from '#data/gahuza/bbc_gahuza_radio/p02pcb5c.json';
+// eslint-disable-next-line import/order
+import routes from '.';
```

Comment:
`import routes from '.';` seems to need to be above the alias imports from below, however the linter doesn't like this.

To get both the linter passing and path aliasing to work I had to use this command `// eslint-disable-next-line import/order`

I looked if there are other instances of this exact command in Simorgh, there isn't. but there are many instances of commands similar that start with `eslint-disable-next-line…`

There is however an instance of a command to disable the import order for a whole file `/* eslint-disable import/order */` in `src/server/utilities/customMetrics/index.test.ts`

The imports from `#app` and `#psammead` in this file specifically didn’t get flagged for needing an alias in Karina's PR #12206  that shows where errors for `Relative import from another package is not allowed.` are. if we don’t want to use this `// eslint-disable-next-line import/order `workaround we i don’t think we have to. Just thought I’d change them to use aliases anyway.

We could leave these imports as they were previously and I think they still might not show up when linting, as they weren't flagged before?

Don’t know how people feel about this?